legacy/Dockerfile

```diff
@@ -27,6 +27,10 @@ RUN { \
 		echo 'opcache.enable_cli=1'; \
 	} > /usr/local/etc/php/conf.d/opcache-recommended.ini
 
+# SQLite Directory Setup
+RUN mkdir -p /var/www/data \
```

Comment:
I'm slightly worried that while it's discouraged, `/var/www` is still sometimes used as a web root.